app/controllers/chat_channels_controller.rb

```diff
@@ -31,7 +31,11 @@ def create
 
   def update
     ChatChannelUpdateService.new(@chat_channel, chat_channel_params).update
-    flash[:settings_notice] = "Channel Settings Updated."
+    if @chat_channel.valid?
+      flash[:settings_notice] = "Channel Settings Updated."
+    else
+      flash[:settings_notice] = "#{@chat_channel.errors.full_messages}"
```

Comment:
What do you think about `join`ing `full_messages` for readability?

Reply:
Using [to_sentence](https://api.rubyonrails.org/classes/Array.html#method-i-to_sentence) it's even better :)

HT to @citizen428 for telling us about this

Reply:
Cool, I still don't know all the rails magic :sparkles: